unittests/test_jira_webhook.py

```diff
@@ -409,46 +409,46 @@ def test_webhook_get(self):
     def test_webhook_jira_disabled(self):
         self.system_settings(enable_jira=False)
         response = self.client.post(reverse('jira_web_hook'))
-        self.assertEqual(404, response.status_code, response.content[:1000])
+        self.assertEqual(201, response.status_code, response.content[:1000])
 
     def test_webhook_disabled(self):
         self.system_settings(enable_jira=False, enable_jira_web_hook=False)
         response = self.client.post(reverse('jira_web_hook'))
-        self.assertEqual(404, response.status_code, response.content[:1000])
+        self.assertEqual(201, response.status_code, response.content[:1000])
 
     def test_webhook_invalid_content_type(self):
         self.system_settings(enable_jira=True, enable_jira_web_hook=True, disable_jira_webhook_secret=True)
         response = self.client.post(reverse('jira_web_hook'))
         # 400 due to incorrect content_type
```

Comment:
For some reason GitHub won't show the right line here, but these `# 4XX` comments should be updated or removed

Reply:
They should be updated from 4xx to 2xx

This comment shows the correct line with the correct change. Resolving